src/assets/default-location.svg

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
Nice default image y'all made, did you design it yourself?

Reply:
Thank you! I would love to create amazing SVGs, but I need to learn more 😄. We got this [icon](https://www.reshot.com/free-svg-icons/item/park-ZSXLFMHRK2/) from Reshot.